test/model-method.js

```diff
@@ -68,7 +67,6 @@ describe('ModelMethod', function() {
       {
         modelId: userModel.id,
         name: 'multiMethod',
-        isStatic: true,
```

Comment:
I find the changes in the existing test suite suspicious. Since we assume LB 2.0 by default, I would expect the current test cases should keep passing without any changes even with the new implementation.